### DISCUSSION TITLE
Unexpected parameters

### DISCUSSION THREAD

**Discussion by getrude386** [mboquien/cigale#54] [Q&A]
Hi,

Please assist in solving this:

KeyError: 'The parameters passed are different from the expected one. Unexpected parameters: alpha_agn, qir_sf, R_agn, alpha_sf. '

Does it mean my cigale version is old?

![Screenshot 2023-05-11 at 11 18 15](https://github.com/mboquien/cigale/assets/105718721/a3401da6-67e7-4de6-bf26-deaadc637331)

**Comment by mboquien** [comment]
I see you are using a development version. Would you have the git revision hash?

**Reply by getrude386** [reply]
805612292d50c8ee0fb124705feb94614b106d7a

**Comment by mboquien** [comment]
Could you delete the `pcigale.ini` and `pcigale.ini.spec` files and redo `pcigale init` and `pcigale genconf` please? I have the feeling that you are using an old `pcigale.ini` with a newer version of cigale.

**Reply by getrude386** [reply]
I did that, and I am still getting the same error.

**Reply by getrude386** [reply]
I also deleted Cigale and re-installed it, and I still get the same error.

**Reply by mboquien** [reply]
Could you send your latest `pcigale.ini` and `pcigale.ini.spec` files? Thanks!

**Reply by getrude386** [reply]
[pcigale.ini (1) copy.txt](https://github.com/mboquien/cigale/files/11515234/pcigale.ini.1.copy.txt)
[pcigale.ini.spec.txt](https://github.com/mboquien/cigale/files/11515235/pcigale.ini.spec.txt)
I attached them.

**Reply by mboquien** [reply]
There seems to be a problem with your `pcigale.ini` file. The `bands` parameter ends with `suba>`, as if truncated. It is the case for the `fracAGN` parameter too. As far as I can tell, your `pcigale.ini.spec` file is also severely malformed, as if copied and pasted from a text editor.

**Reply by getrude386** [reply]
Hi
[pcigale.ini.spec.txt](https://github.com/mboquien/cigale/files/11584808/pcigale.ini.spec.txt)
[pcigale.ini.txt](https://github.com/mboquien/cigale/files/11584809/pcigale.ini.txt)

I did copy and paste, apologies. I have now attached the correct files.

**Reply by mboquien** [reply]
I tried it and it seems to work fine here. My best guess is that you have somehow conflicting installations. Is it possible you have generated the `pcigale.ini` on one computer and are trying to run it on another for instance?

**Reply by getrude386** [reply]
I just realized that it works when I run it through an interactive session. I get that error only when I submit a slurm job in a cloud that I am using. Maybe the pcigale version in the cloud is an old version.
